server/terraform/modules/myhealth/main.tf

```diff
@@ -69,6 +69,7 @@ resource "google_project_service" "service" {
     "compute.googleapis.com",
     "dns.googleapis.com",
     "firebase.googleapis.com",
+    var.backend_version == "v2" ? "firebasehosting.googleapis.com" : null,
```

Comment:
I wonder if it's clearner to simply fork this to myhelathv2/main.tf. May be much cleaner to manages.

Reply:
Done.